plugins/modules/ec2_snapshot.py

```diff
@@ -70,6 +70,38 @@
     required: false
     default: 0
     type: int
+  modify_create_vol_permission:
+    description:
+      - If set to C(true), ec2 snapshot's createVolumePermissions can be modified.
+    required: false
+    type: bool
+    version_added: 6.0.0
+  purge_create_vol_permission:
+    description:
+      - Whether unspecified group names or user IDs should be removed from the snapshot createVolumePermission.
+      - Must set I(modify_create_vol_permission) to C(True) for when I(purge_create_vol_permission) is set to I(purge_create_vol_permission).
```

Comment:
> for when I(purge_create_vol_permission) is set to I(purge_create_vol_permission)

This doesn't make sense to me. Do you mean when purge_create_vol_permission is set to true?

Reply:
fixed.